docs/getting-started.md

```
@@ -0,0 +1,100 @@
+# Getting Started
+
+Welcome to the Getting Started guide for the refarch-templates. This guide
+is divided into sections for Frontend/Web Component and Backend/EAI, as well
+as a general setup and requirements applicable to all templates.
+
+## General
+
+To use the refarch-templates, you need to have Docker installed on your
+machine. Inside the `stack` folder, you will find a `docker-compose.yml`
+file that will spin up everything needed for local development. 
```

Markdownlint (0.37.0)

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)